lib/index.js

```diff
@@ -78,6 +84,34 @@ const init = async function (options = {}) {
     protocols: shortstopHandlers
   })
 
+  // const tmpDir = Path.join(__dirname, '.tmp_' + Date.now())
+  // const destination = join(tmpDir, 'output')
+
+  await server.register({
```

Comment:
This makes this plugin unconfigurable and not what we want to push out as the default.  I know there were some issues getting pretty printing working; those should be handled in a different manner that retains the flexibility above.